### PR TITLE
Fix/changes in meeting availability - #300

### PR DESCRIPTION
**Pull request by SlakvichAleksandr**
https://prnt.sc/NWwb24abol2B - result

### CODE REVIEW

File: frontend/src/app/modules/availability/team/team.component.html [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+                <img class="avatar" src="assets/header/avatar-icon.png" />
+            </ng-template>
+        </div>
         <div class="team-name">
             <div>
                 <p>{{ teamName }}</p>
             </div>
         </div>
-        <img src="assets/copy_icon.png" class="copy-icon" />
     </div>
```
Comment (YuraRiabov) [inline frontend/src/app/modules/availability/team/team.component.html:16]: Shouldn't we have team link here?

**Review by YuraRiabov — CHANGES_REQUESTED** [review]

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs [hunk 2]
```diff
@@ -66,11 +68,13 @@ public async Task<UserPersonalAndTeamSlotsDto> GetUserPersonalAndTeamSlotsAsync(
...
                 .Where(x => x.Type == SlotType.Team)
-                .GroupBy(x => x.TeamName)
+                .GroupBy(x => x.TeamId)
                 .Select(x =>
                     new AvailabilitySlotsGroupByTeamsDto
                     {
-                        Name = x.Key,
+                        Id = x.Key,
+                        Name = x.First().TeamName,
+                        Image = x.First().TeamLogoPath,
```
Comment (tatianahutii) [inline backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs]: You can group by all Team or by 3 properties to avoid using x.First()

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/availability/team/team.component.html (hunk 1 above)
Comment (SlakvichAleksandr) [reply frontend/src/app/modules/availability/team/team.component.html:16]: https://prnt.sc/scqELaT5lQAL

Comment (YuraRiabov) [reply frontend/src/app/modules/availability/team/team.component.html:16]: Interesting

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs (hunk 2 above)
Comment (SlakvichAleksandr) [reply backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs]: we make select from dto model that doesn't have Team entity
https://prnt.sc/dW6xooMygVJy

Comment (tatianahutii) [reply backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs]: Ok. than you can use the second option

Comment (SlakvichAleksandr) [reply backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs]: fix by multiple columns. thanks )

**Review by tatianahutii — APPROVED** [review]

**Review by YuraRiabov — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**